pkg/runner/runner.go

```diff
@@ -80,7 +82,10 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	envy.Parse("PROMSCALE")
 
 	// Ignore errors; CommandLine is set for ExitOnError.
-	_ = flag.CommandLine.Parse(args)
+	_ = ff.Parse(flag.CommandLine, args,
+		ff.WithConfigFile("config.yml"),
```

Comment:
Shouldn't this name be supplied from CLI?

Reply:
Yeah I think using `WithConfigFileFlag` is better and define a new flag with `config.yml` as the default value

Reply:
Yes, I simplified it for the draft, will add a flag for the filename.